CHANGELOG.md

```diff
@@ -5,27 +5,39 @@ CHANGELOG
 -----
 
 **ENHANCEMENTS**
-- Add support for updating `SharedStorage` configuration.
+- Add support for AWS Trainium instances.
+- Add support for Slurm Accounting.
+- Add support for adding and removing shared storages at cluster update by updating `SharedStorage` configuration.
 - Add new configuration parameter `DeletionPolicy` for EFS and FSx for Lustre shared storage
   to support storage retention on deletion.
-- Enable server-side encryption for the EcrImageBuilder SNS topic created when deploying ParallelCluster API and used to notify on docker image build events.
+- Add possibility to specify multiple instance types for the same compute resource.
 - Add support for on-demand capacity reservations.
-- Add support for specifying the supported IMDS version in cluster and build image configurations via the `Imds > ImdsSettings` property. A value of `v1.0` (the default) means that both IMDSv1 and IMDSv2 are supported, while a value of `v2.0` means that only IMDSv2 is supported.
-- Add support for Slurm Accounting.
+- Enable server-side encryption for the EcrImageBuilder SNS topic created when deploying ParallelCluster API and used to notify on docker image build events.
+- Add support for specifying the supported IMDS version in cluster and build image configurations via the `Imds/ImdsSettings` property.
+  A value of `v1.0` (the default) means that both IMDSv1 and IMDSv2 are supported, while a value of `v2.0` means that only IMDSv2 is supported.
```

Comment:
I'd avoid adding the details of the parameter or accepted value. I'd remove the "A value ... " sentence. 
This should stay in the documentation not in the changelog.